### PR TITLE
update mailgun readme

### LINKED ISSUE #8492: miss important things on strapi-provider-email-mailgun documentation
### Describe the bug

If you are using EU servers, you need to specify the host. And it's not mentioned in the documentation.

### Steps to reproduce the behavior

only need to add host as a property on `providerOptions`

### Expected behavior

Users can send and receive emails using mailgun, and it's not working correctly with the info from the documentation.

### Code snippets

```
module.exports = ({ env }) => ({
  // ...
  email: {
    provider: 'mailgun',
    providerOptions: {
      apiKey: env('MAILGUN_API_KEY'),
      domain: env('MAILGUN_DOMAIN'), //Required if you have an account with multiple domains
      host: env('MAILGUN_HOST') // we have only 2 options here. api.mailgun.net for US servers, api.eu.mailgun.net for EU Servers.
    },
    settings: {
      defaultFrom: 'myemail@protonmail.com',
      defaultReplyTo: 'myemail@protonmail.com',
    },
  },
  // ...
});
```

### System

- Node.js version: 10.16.0
- NPM version: 6.9.0
- Strapi version:  3.2.5
- Database: MySQL
- Operating system: using WSL

### PR DESCRIPTION
**Pull request by andyatflocc**
fix #8492 
Ready to be merged.

### What does it do?

One additional line of example config 

### Why is it needed?

If you use mailgun's EU servers, without this one line, the plugin does not work

### Related issue(s)/PR(s)
https://github.com/strapi/strapi/issues/8492
Let us know if this is related to any issue/pull request

### CODE REVIEW

File: packages/strapi-provider-email-mailgun/README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ module.exports = ({ env }) => ({
     providerOptions: {
       apiKey: env('MAILGUN_API_KEY'),
       domain: env('MAILGUN_DOMAIN'), //Required if you have an account with multiple domains
+      host: env('MAILGUN_HOST') // If you are using EU servers, you need to specify the host. api.eu.mailgun.net for EU Servers (or api.mailgun.net for US servers)
```
Comment (derrickmehaffy) [inline packages/strapi-provider-email-mailgun/README.md:49]: If this is optional it needs to have the default value in the env function. 

`env('MAILGUN_HOST', 'whatever the default is')`

**Review by derrickmehaffy — CHANGES_REQUESTED** [review]
Please also set the source branch and target branch for the PR as `documentation` and not the `master` branch.

https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#documentation

**Comment by alexandrebodin** [comment]
Hi, you branch is out of sync with the repo. I think you started from master & switched to documentation but you brought with it a few commits. can you make sure your PR only contains what you did ?

**Comment by alexandrebodin** [comment]
Hello, I'm going to close this PR because of inactivity. Feel free to open a new on or ping me here if you want to finish it ;)

**Pull request closed without merging**